lib/RedditScraper.ts

```diff
@@ -70,10 +70,13 @@ export class RedditScraper {
 			},
 		};
 
+		requestOptions.qs = {};
+		
+		if (options.Records) {
+			requestOptions.qs["limit"] = options.Records;
+		}
 		if (options.FullName) {
-			requestOptions.qs = {
-				after: options.FullName,
-			};
+			requestOptions.qs["after"] = options.FullName;
```

Comment:
Similar to the above

```suggestion
			requestOptions.qs.after. = options.FullName;
```